specter-serialization/src/main/java/dev/spiritstudios/specter/api/serialization/Commentable.java

```diff
@@ -0,0 +1,9 @@
+package dev.spiritstudios.specter.api.serialization;
+
+/**
+ * Represents an object that can have comments.
+ */
+public interface Commentable {
+	void setComments(String... comments);
+	String[] comments();
```

Comment:
arrays can be a pain in the ass down the line. I suggest not using them except for extremely low level code.

Reply:
fixed